include/ebpf_result.h

```diff
@@ -97,6 +97,9 @@ extern "C"
 
         /// Caller has reached tail call limit.
         EBPF_NO_MORE_TAIL_CALLS,
+
+        // The container can not hold additional elements.
+        EBPF_OUT_OF_SPACE,
```

Comment:
why not EBPF_INSUFFICIENT_BUFFER?
And does this map to a specific ntstatus in _ebpf_result_to_ntstatus ?